data/wordpressa/santuario-xml-security1/project.yml

```diff
@@ -0,0 +1,5 @@
+name: Apache Qpid
+repository:
+  type: git 
+  url: https://github.com/apache/qpid-broker-j.git
+Url: https://github.com/apache/qpid-broker-j.git.   #https://qpid.apache.org
```

Comment:
Why the dot after the URL and the comment?

Reply:
This is fixed with commit 5055704.